### ISSUE TITLE
please replace Twitter Bootstrap 2.x with Bootstrap 3.x

### ISSUE THREAD

**Issue by jonassmedegaard** [doctrine/doctrine-sphinx-theme#6]
Twitter Bootstrap 2.x is dead upstream.

Please migrate to Bootstrap 3.x: https://getbootstrap.com/migration/

**Comment by Ocramius** [comment]
@jonassmedegaard there's no need for changing this unless we want to change the design of the site...

**Comment by jonassmedegaard** [comment]
so you see no problem in exposing to your users javascript code which noone maintains regarding security flaws or other problems?

**Comment by Ocramius** [comment]
@jonassmedegaard no, mainly because it's static pages with no possible user input except for disqus, which is in an iframe.

As far as I can see, we may even be able to strip down the remaining JS instead of upgrading.

**Comment by jonassmedegaard** [comment]
that'd be a sensible approach too - my worry is your distributing code that is unmaintained, not specifically to a solution of pushing to newer Bootstrap (sorry, I realize my initial post was misleading).

**Comment by Ocramius** [comment]
Yeah, the fact is that this is not really for redistributing: it's just
some rudimentary template for the project site :-P

**Comment by jonassmedegaard** [comment]
Quoting Marco Pivetta (2015-10-17 18:05:49)
> Yeah, the fact is that this is not really for redistributing: it's 
> just some rudimentary template for the project site :-P

It seems to me that it is used not only in your website design but also 
in the documentation of the ORM component: 
https://packages.debian.org/jessie/doctrine-orm-doc

 - Jonas

-- 
 * Jonas Smedegaard - idealist & Internet-arkitekt
 * Tlf.: +45 40843136  Website: http://dr.jones.dk/

 [x] quote me freely  [ ] ask before reusing  [ ] keep private

**Comment by Ocramius** [comment]
Why is there even a package with the ORM docs? The docs are just supposed
to be downloadable as PDF if needed :-\

Marco Pivetta

http://twitter.com/Ocramius

http://ocramius.github.com/
